data_prep/generate_climos.py

```diff
@@ -90,45 +61,43 @@ def create_climo_file(fp_in, fp_out, t_start, t_end, variable):
         'tasmax', 'pr'
     }
 
-    if variable not in supported_vars:
-        raise Exception("Unsupported variable: cant't yet process {}".format(variable))
-
-    # Allow different ops by variable? # op = 'sum' if variable == 'pr' else 'mean'
```

Comment:
Perhaps my transfer of this into the docstring for `climo_outputs` (line 69) wasn't clear enough. But I did not intend to remove that information.

That said, it's worth checking with scientists. Trevor is coming by to talk about processing the climdex variables, I'll add this to the agenda.